src/components/contact_us/components/send_mail_form/sendForm.less

```diff
@@ -70,8 +70,12 @@
 	color: var(--white-color)
 }
 .not_valid {
-	border-color: red !important;
+	border-color: var(--not-valid-color) !important;
 }
 .not_valid_label {
-	color: red;
+	color: var(--not-valid-color);
+}
+
+body[style="direction: rtl;"] & {
```

Comment:
When I was young relying on such plain strings was not the best idea - especially if the strings contain white-spaces and punctuation marks  
Setting a `.rtl` class seems more safe

```css
	componentDidMount () {
		document.body.classList.add(config.isRTL ? 'rtl' : 'ltr')
	}
```
https://github.com/Doc999tor/Mobile-Atzmaim-website/blob/0a9bcb08331f8ab9011fd092f17fb65af7bb8f4f/src/components/app.js#L16